resources/[race]/race_progress/client.lua

```diff
@@ -788,7 +791,7 @@ function drawText(text,x,y,color,backgroundColor)
 		dxDrawLine(x+textWidth+cornerSize/2,y-1,x+textWidth+cornerSize/2,y+fontHeight,tocolor(0,0,0,255))
 		]]
 	end
-	dxDrawText(text,x,y,x,y,color,fontScale,font)
+	dxDrawText(text,x,y,x,y,color,fontScale,font,_, _, false, false, false, true, false)
```

Comment:
Are you sure this works? I think the _ _ need to be strings
But if you tested it I don't mind